Unify layout of all ContentObjects Part 1 (#565)
TYPO3-Documentation/TYPO3CMS-Reference-Typoscript: Documentation/ContentObjects/Files/Index.rst

FILES
=====

> A content object of type FILES was integrated with the File Abstraction Layer

Reviewer: ```suggestion
A content object of type FILES uses the File Abstraction Layer
```

> (FAL) and is there to output information about files.

Reviewer: ```suggestion
(FAL) and is used to display information about files.
```

.. _cobj-files-examples:

> Examples:

Reviewer: ```suggestion
Examples
```

> ==========

Reviewer: ```suggestion
=========
```
